### ISSUE TITLE
Iphone X safe area when status bar is off

### ISSUE THREAD

**Issue by packytagliaferro** [EddyVerbruggen/nativescript-feedback#39]
Currently the feedback is cutoff on iPhone X if you also set the status bar to hidden:

info.plst

```
<key>UIStatusBarHidden</key>
		<true/>
		<key>UIViewControllerBasedStatusBarAppearance</key>
		<false/>
```

result:
![screen shot 2018-12-05 at 4 29 59 pm](https://user-images.githubusercontent.com/1316850/49553302-fb0f9a80-f8ab-11e8-9028-ea869d143bb8.png)

**Comment by EddyVerbruggen** [comment]
Ah, interesting. I think you're the first person to use this plugin with a hidden statusbar 😉

**Comment by EddyVerbruggen** [comment]
Fix available in 1.3.2 from npm. Thanks for reporting!
